deploy/letsencrypt-template.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: letsencrypt-template
+metadata:
+  annotations:
+    openshift.io/display-name: Let's Encrypt,
+    description: Template for Deploying a Let's Encrypt Controller.
+    tags: infrastructure,letsencrypt,controller
+    template.openshift.io/documentation-url: https://github.com/tnozicka/openshift-acme.git
+  name: letsencrypt
+objects:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: ${SERVICEACCOUNT_NAME}
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: ${NAME}
+  spec:
+    selector:
+      app: ${NAME}
+    ports:
+    - name: http
+      protocol: TCP 
+      port: 80
+      targetPort: 5000
+- kind: Deployment
+  apiVersion: extensions/v1beta1
```

Comment:
Deployments now live in apps (I think since 1.6), v1beta1 is probably ok because v1beta2 is 1.8 only
Do you plan on supporting older than Kube 1.6?